### PR TITLE
Phantom local files

### PR DESCRIPTION
**Pull request by giakki**
@XhmikosR I think there is little we can do about this issue.
if you run `bin/uncss tests/phantomjs/local_script.html --htmlroot="tests/phantomjs"` you can see the problem is that both absolute-sourced scripts and relative ones have the same prefix, therefore it's impossible to differentiate between the two :/

### CODE REVIEW

**Comment by giakki** [comment]
@XhmikosR I think there is little we can do about this issue.
if you run `bin/uncss tests/phantomjs/local_script.html --htmlroot="tests/phantomjs"` you can see the problem is that both absolute-sourced scripts and relative ones have the same prefix, therefore it's impossible to differentiate between the two :/

File: src/phantom.js [hunk 1]
```diff
@@ -71,6 +72,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
                 /* Protocol-less URL */
                 url = 'http://' + originalUrl.substr(7);
             }
+            console.log('After: ' + url);
```
Comment (RobLoach) [inline src/phantom.js]: Debugging information? There's a couple console.log()'s in there... Should either remove them, or use `require('debug')`, or another logging method.

**Comment by XhmikosR** [comment]
@giakki: is this still an issue?

**Comment by XhmikosR** [comment]
Closing this since we switched to jsdom.

**Pull request closed without merging**